src/components/Header.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+import Time from './Time';
+
+const Header = () => (
+    <header className="header">
+        <Time />
+        <h1 className="title">CALORIES COUNTER</h1>
```

Comment:
it is better to put such an important heading (h1) as first one, to improve positioning app in google (goole bots will read at first this title, so SEO will be better). It shouldn't influence styling, as in `<Time />` component we are positioning at by `absolute`. Check it out